app/helpers/search_helper.rb

```diff
@@ -8,4 +8,69 @@ def view_more_text
       'website'
     end
   end
+
+  def access_level
+    if guest?
+      'guest'
+    else
+      'full'
+    end
+  end
+
+  # Current request IP is outside our campus range
+  def guest?
```

Comment:
This was really built for use in the EDS API calls to send along with the API call. I think `guest?` will make that call clearer than `!member?`. `access_level` was a last minute addition and is possibly of no use at all.